deploy/docker-entrypoint.d/10-wait-for-postgres.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 if [ -n "${POSTGRE_HOST:-}" ]; then
   echo >&3 "=> Waiting for postgres..."
-  until PGPASSWORD=$POSTGRE_PASSWORD psql -h "$POSTGRE_HOST" -p $POSTGRE_PORT -U "$POSTGRE_USER" -c '\q'; do
+  until PGPASSWORD=$POSTGRE_PASSWORD psql -h "$POSTGRE_HOST" -p $POSTGRE_PORT -U "$POSTGRE_USER" -d "${POSTGRE_NAME:=root}" -c '\q'; do
```

Comment:
POSTGRE_NAME:=postgres
because it should be in sync with https://github.com/heartexlabs/label-studio/blob/master/docker-compose.yml#L33